examples/eris/src/commands/hello.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+  name: "hello",
+  description: "Says hello!",
+  async execute(message, args) {
+    message.channel.createMessage(
+      "Hello " + message.author.username + "!" + " Choo choo 🚅",
+    );
```

Comment:
```suggestion
    message.channel.createMessage(`Hello ${message.author.username}!, Choo choo 🚅`);
```

Dont indent and use [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals).